docs/web/tutorial-common-lifecycle-hooks.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Lifecycle hooks for starting Workflows.
+description: VertiGIS Studio Web - Learn what lifecycle hooks are available and when to use them.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import UseCaseCard from "../../src/components/UseCaseCard";
+import UseCaseContainer from "../../src/components/UseCaseContainer";
+
+This article will tell you about common lifecycle hooks within Web and when you should use them for starting workflows.
+
+# Services > App > Events >
+
+## Application Initializing
+
+Event gets executed when Web begins to initialize. Commands or workflows must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading. This can be useful for launching workflows that are going to impact the initial state of the application (such as initial layout).
+
+## Application Initialized
+
+Event gets executed when Web has initialized. Commands or workflows must not contain blocking actions such as alerts or user-facing forms; these will prevent the application from loading in Designer.
+
+# Components > Map > Events >
+
+## Map Initialized
+
+Event that gets triggered when the map is initialized. The map is initialized when it is loaded for the first time, a user signs in or out, or a map area is activated or deactivated. This can be useful for displaying a workflow that presents UI such as a form or alert or prompt.
+
+# Services > Launch Link Workflows
+
+Using the Launch Link service you can add and remove workflows to the app that can be optionally specified and run by URL parameter. The launch link workflow is run at the same time as the Map Initialized Event.
```

Comment:
Screenshot showing where in Designer to add the Launch Link Workflows? You do have a screenshot of the deploy dialog stuff but I mean the step before that where you make the workflows available for launch links.